config/config-default/core.entity_form_display.node.document.default.yml

```diff
@@ -252,14 +252,21 @@ content:
     type: metatag_firehose
     region: content
   field_subject_matter:
-    weight: 9
+    weight: 8
     settings:
       match_operator: CONTAINS
       size: 60
-      match_limit: 10
-      placeholder: ''
+      autocomplete_route_name: autocomplete_deluxe.autocomplete
+      selection_handler: default
+      limit: 10
+      min_length: 0
+      delimiter: ''
+      not_found_message_allow: false
+      not_found_message: 'The term ''@term'' will be added'
+      new_terms: false
+      no_empty_message: 'No terms could be found. Please type in order to add a new term.'
     third_party_settings: {  }
-    type: super_term_reference_autocomplete_widget
```

Comment:
@melwong-jcc can you rebase and lando fresh from master?  this line should not be overwritten.

Reply:
Ack! Will redo.